src/main/frontend/handler/global_config.cljs

```diff
@@ -58,8 +60,11 @@
 - Start a file watcher for global config dir if it's not already started.
   Watcher ensures client db is seeded with correct file data."
   [{:keys [repo]}]
-  (p/let [root-dir' (ipc/ipc "getLogseqDotDirRoot")
-          _ (reset! root-dir root-dir')
-          _ (restore-global-config!)
-          _ (create-global-config-file-if-not-exists repo)
-          _ (fs/watch-dir! (global-config-dir) {:global-dir true})]))
+  (-> (p/let [root-dir' (ipc/ipc "getLogseqDotDirRoot")
```

Comment:
electron/ipc should not happen on mobile platforms

Reply:
`start` shouldn't be called on mobile platforms but if it is, steps to reproduce would be helpful. Having `electron.ipc` in so many frontend namespaces is confusing and gives the misleading impression that our frontend has electron main process code. I think it'd be helpful if we renamed this ns to something that doesn't start with `electron` e.g. `frontend.electron.ipc`